packages/account/src/Components/additional-kyc-info-modal/additional-kyc-info-modal.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { observer, useStore } from '@deriv/stores';
+import {
+    Modal,
+    DesktopWrapper,
+    MobileWrapper,
+    UILoader,
+    PageOverlay,
+    Div100vhContainer,
+    Text,
+    InlineMessage,
+} from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import AdditionalKycInfoForm from './additional-kyc-info-form';
+
+type TAdditionalKycInfoFormWithHintBox = {
+    error?: unknown;
+    setError?: React.Dispatch<React.SetStateAction<unknown>>;
+};
+
+const AdditionalKycInfoFormWithHintBox = ({ error, setError }: TAdditionalKycInfoFormWithHintBox) => {
+    return (
+        <React.Fragment>
+            {!!error && (
+                <InlineMessage
+                    size='sm'
+                    message={
+                        error &&
+                        typeof error === 'object' &&
+                        'message' in error &&
+                        typeof error.message === 'string' &&
+                        error.message
+                    }
+                    type='error'
+                />
+            )}
+            <AdditionalKycInfoForm setError={setError} />
+        </React.Fragment>
+    );
+};
+
+export const AdditionalKycInfoModal = observer(() => {
+    const {
+        ui: { is_additional_kyc_info_modal_open: is_open, toggleAdditionalKycInfoModal },
+    } = useStore();
+    const [error, setError] = React.useState<unknown>('');
+
+    const toggleModal = (e?: React.MouseEvent<HTMLElement, MouseEvent> | undefined) => {
+        // if e.target is anchor tag, don't close modal for link click within modal
+        const target = e?.target as HTMLElement;
+        if (target.tagName === 'A') e?.stopPropagation();
```

Comment:
Interesting logic. Can this be handled by passing anchor tag for `elements_to_ignore` prop of Modal component?

Reply:
Did try yet wasn't able to achieve the expected behaviour